lms/djangoapps/certificates/views/webview.py

```diff
@@ -527,17 +536,26 @@ def render_html_view(request, user_id, course_id):
         course_key = CourseKey.from_string(course_id)
         user = User.objects.get(id=user_id)
         course = modulestore().get_course(course_key)
-        national_id = NationalId.objects.get(user=user_id)
 
     # For any other expected exceptions, kick the user back to the "Invalid" screen
-    except (InvalidKeyError, ItemNotFoundError, User.DoesNotExist, NationalId.DoesNotExist) as exception:
+    except (InvalidKeyError, ItemNotFoundError, User.DoesNotExist) as exception:
         error_str = (
             "Invalid cert: error finding course %s or user with id "
             "%d. Specific error: %s"
         )
         log.info(error_str, course_id, user_id, str(exception))
         return render_to_response(invalid_template_path, context)
 
+    # Get National number
+    national_id = _get_user_national_id(user, preview_mode)
+    if not national_id:
+        log.info(
+            "Invalid cert: User %s does not have national identity number.",
```

Comment:
@pepeportela This log should only be shown when not in `preview_mode`. Instead of `info` I would use `error` or `warning`.